gcsusrc.c

```diff
@@ -1326,7 +1216,8 @@ int main(int argc, char **argv)
 			} else {
 				sprintf(message, "message %lu", count);
 			}
-			exinfo.ume_msg_clientd = (void *)((lbm_uint_t)count + 1);
+			exinfo.ume_msg_clientd = (void *)((long long)count + 1);
```

Comment:
1. Was this change a fix to a bug?
2. I've never liked commenting out old code. Git does a fine job of showing you older code.
3. One of the C compilers, I don't remember which one, doesn't like double-slash comments for standard C.

Reply:
This was to get rid of a compiler warning. I took out the commented-out old code.